app/assets/stylesheets/modules/_modules.styl

```diff
@@ -122,3 +122,7 @@
     +below(desktop)
       margin-top: 0px
 
+.course-alerts-wrapper
+  +above(desktop)
+    margin-top: 0px !important
```

Comment:
What is overriding this when !important is not used?

Reply:
On desktop mode it was receiving a height using inline css which wasn't required